packages/app/src/cli/models/extensions/specification.ts

```diff
@@ -219,13 +266,14 @@ export interface CreateExtensionSpecType<TConfiguration extends BaseConfigConten
  * hasExtensionPointTarget?: (configuration: TConfiguration, target: string) => boolean // function to determine if the extension has a given extension point target
  * ```
  */
-export function createUIExtensionSpecification<TConfiguration extends BaseConfigContents = BaseConfigContents>(
+export function createExtensionSpecification<TConfiguration extends BaseConfigContents = BaseConfigContents>(
   spec: CreateExtensionSpecType<TConfiguration>,
-): UIExtensionSpec<TConfiguration> {
+): ExtensionSpecification<TConfiguration> {
   const defaults = {
     // these two fields are going to be overridden by the extension specification API response,
     // but we need them to have a default value for tests
     externalIdentifier: `${spec.identifier}_external`,
+    additionalIdentifiers: [],
```

Comment:
it's not immediately clear that additionalIdentifiers contains the names of all variants of the extension. maybe we could call it `variantIdentifiers`? I dunno!

Reply:
I added this just for functions, because we will be migrating from multiple identifiers (order_discount, product...) to a single identifier: `function` and we need to be compatible with both cases, but there is a single specification for all